plugins/confluence/commands/search.md

```
@@ -0,0 +1,121 @@
+---
+description: Search Confluence for related documentation using natural language or CQL
+argument-hint: "<query> [--space <key>] [--limit <n>]"
+---
+
+## Name
+confluence:search
+
+## Synopsis
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)